userguide/dailyuse/waydroid.rst

```diff
@@ -0,0 +1,81 @@
+Android apps
+========================
+
+`Waydroid <https://waydro.id>`_ is a minimal Android container and compatibility layer to run Android apps on GNU/Linux operating systems such as Ubuntu Touch.
+
+.. note::
+    "Computer" refers to another device you connect your Ubuntu Touch device to (via USB here).
+    Your USB-attached computer must have ``adb`` and ``fastboot`` installed.
+
+.. note::
+    You will need to execute commands on your Ubuntu Touch device (and/or attached computer) to initialize Waydroid and install APKs.
+    You can do that by using the terminal application, but it is easier to use ``adb shell`` or :doc:`set up SSH </userguide/advanceduse/ssh>` to access your Ubuntu Touch device from your computer.
+
+
+How to set up
+-------------
+
+All Halium 9 and newer devices ship with waydroid out of the box starting Ubuntu Touch focal, if your device fits these requirements you can follow the folowing steps
+
+#. Run ``adb shell`` from your computer to get a shell on your Ubuntu Touch device (or open the terminal app on your ubuntu touch device).
+#. Run the following command on your Ubuntu Touch device: ``waydroid init``.
+
+Now you're done! You might have to refresh the app drawer (pull down from the center of the screen and release) for the new Android apps to show up.
+
+How to install new APKs
+-----------------------
+
+- Copy the APK to ``/home/phablet/Downloads``. Then run the following from your computer::
+
+    adb shell # Connect to your Ubuntu Touch device. You can also simply open the terminal app on your device.
+    waydroid app install /home/phablet/Downloads/my-app.apk
+    exit
+
+- Done! You might have to refresh the apps scope (pull down from the center of the screen and release) for the new Android apps to show up.
+
+Keep your apps up to date
+-------------------------
+
+- To keep your apps up to date you can use of F-Droid or Amazon Appstore. If you want to install any of the above apps you can find them here:
+
+- F-Droid: https://f-droid.org/
+- Amazon Appstore: https://www.amazon.com/gp/mas/get-appstore/android
+
+How to uninstall apps
+---------------------
+
+- To uninstall apps from the Ubuntu Touch device, run ``adb uninstall [APP_ID]`` from your computer::
```

Comment:
aaand also here, please use the same style for this set of instructions as the two above

Reply:
made it more consistent